polyfill.js

```diff
@@ -20,7 +20,25 @@
   } catch (e) {
     var CustomEvent = function(event, params) {
       var evt, origPrevent;
-      params = params || {};
+
+      // We use here some version of `Object.assign` implementation, to create a shallow copy of `params`.
+      // Based on https://github.com/christiansany/object-assign-polyfill/blob/213cc63df14515fb543117059d1576204bfaa8a7/index.js
+      params = (function(from) {
+        'use strict';
+
+        var to = {};
+        // Skip over if undefined or null
+        if (from != null) {
```

Comment:
This shouldn't be required since you're ensuring the passed parameter is never null in L40 below (`params || {}`)

Reply:
It was just a simple optimization in my opinion, but I agree.
Do you want to change L40 instead to not contain `|| {}` since we're checking it already?